examples/class-names-dynamic-object.tsx

```diff
@@ -5,7 +5,7 @@ export default {
   title: 'class names dynamic object',
 };
 
-export const objectLiteral = () => {
+export const ObjectLiteral = () => {
```

Comment:
Not able to use hooks in 'non components' 
The rule checks to see if the component starts with an uppercase letter. It will throw otherwise

Reply:
hehe i was wondering!